dti-website-redesign/src/components/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import styles from './Footer.module.css';
+
+type Icon = {
+  link: string;
+  src: string;
+  alt: string;
+};
+
+const icons: Icon[] = [
+  {
+    src: '/static/icons/app-store.png',
+    link: 'https://www.example.com',
+    alt: 'app store icon'
+  },
+  {
+    src: '/static/icons/email.png',
+    link: 'emailto: hello@cornelldti.org',
```

Comment:
I think this should be `mailto:hello@cornelldti.org`. Currently theres this error:
<img width="403" alt="image" src="https://user-images.githubusercontent.com/81320443/175861702-b5f93d03-db95-40be-8a8a-5d64f8bcf380.png">
and clicking the icon tries to take you to another page. 

This "mailto" thing also opens some default email app (e.g. not gmail) and picks a random email address to send it from. Maybe this is just on my end, but do you think it would be a better user experience if it just copied the email address to your clipboard or something?

Reply:
Hmm not sure... I feel like that's the typical behavior of clicking on an email icon. I think if we did do something like copy it to the clipboard there'd need to be some kind of feedback to the user that the action happened which might detract from the website. We can always see what Tracy thinks about it.